.gitmodules

```diff
@@ -40,3 +40,6 @@
 [submodule "vendor/status-lib"]
 	path = vendor/status-lib
 	url = https://github.com/status-im/status-lib
+[submodule "spellchecking"]
+	path = spellchecking
```

Comment:
Is there a reason for not putting it inside `vendor/`?

Reply:
sorry but for me `vendor` means like 3rdparty, if you think than its should be in vendor/ i will move it.

Reply:
That makes sense. No need to move it.